### PR TITLE
Spring boot

### PR DESCRIPTION
**Pull request by cyrillbolliger**
- Change infrastructure for spring boot
- Adapt application

### CODE REVIEW

File: camp/pom.xml [hunk 1]
```diff
@@ -15,49 +15,29 @@
...
-            <artifactId>log4j</artifactId>
-            <version>1.2.17</version>
-        </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-            <version>1.7.25</version>
+            <scope>runtime</scope>
         </dependency>
+<!--        <dependency>-->
```
Comment (martyschaer) [inline camp/pom.xml]: Commented out dependency?

File: camp/pom.xml [hunk 2]
```diff
@@ -70,7 +50,7 @@
                 <configuration>
                     <archive>
                         <manifest>
-                            <mainClass>ch.bfh.zodnrk.camp.Application</mainClass>
+                            <mainClass>ch.bfh.zodnrk.camp.DemoApplication</mainClass>
```
Comment (martyschaer) [inline camp/pom.xml]: Should probably not be called 'DemoApplication'.
```suggestion
                            <mainClass>ch.bfh.zodnrk.camp.Application</mainClass>
```

**Review by martyschaer — APPROVED** [review]
Looks mostly good to me!

File: camp/pom.xml (hunk 1 above)
Comment (cyrillbolliger) [reply camp/pom.xml]: Thanks for spotting. It was a legacy from the database switching task...

File: camp/pom.xml (hunk 2 above)
Comment (cyrillbolliger) [reply camp/pom.xml]: Thanks. Fixed as well.

**Pull request merged**